integration/fuse/bin/alluxio-fuse-sdk

```diff
@@ -187,7 +187,7 @@ launch_fuse_process() {
        alluxio.fuse.AlluxioFuse -m ${mount_point} -u ${ufs_address} ${mount_options}"
 
   if [[ "${foreground}" == 'true' ]]; then
-    exec "${cmd}"
+    exec ${cmd}
```

Comment:
is the double quote removed by intention?

Reply:
Yeah, with double quote, the command will fail which is strange, saying that no such command